src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,122 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    private static final int DEFAULT_CAPASITY = 10;
+    private static final int NOT_FOUND = -1;
+    private int capasity;
```

Comment:
misspelled - capaCity, fix everywhere
remove capacity field, it doesn't have any state on its own - we can always retrieve it - array.length

Reply:
\+